Source/WebCore/testing/Internals.mm

```diff
@@ -273,4 +275,32 @@ - (NSAttributedString *)_attributedStringForRange:(NSRange)range
 
 #endif // ENABLE(IMAGE_ANALYSIS_ENHANCEMENTS)
 
+#if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
+bool Internals::emitWebCoreLogs(unsigned logCount, bool useMainThread) const
+{
+    auto blockPtr = makeBlockPtr([logCount] {
+        for (unsigned i = 0; i < logCount; i++)
+            RELEASE_LOG_FORWARDABLE(Media, HTMLMEDIAELEMENT_CREATEMEDIAPLAYER, 0ull);
```

Comment:
Can we define a forward log with `Testing` category and then add `LogStream::testingLogCount` to count only Testing category? This should make the API test more accurate (since we could wait count to reach 1024).